draft-ietf-secevent-http-push.xml

```diff
@@ -131,89 +126,84 @@
+                The mechanisms by which the SET Recipient performs this validation
+                are out of scope for this document. SET parsing and issuer and
                 audience identification are defined in <xref target="RFC8417"/>.
                 The mechanism for validating the authenticity of a SET is implementation
                 specific, and may vary depending on the authentication mechanisms in
                 use, and whether the SET is signed and/or encrypted (See <xref target="aa"/>).
             </t>
             <t>
-                The SET Receiver SHOULD ensure that the SET is persisted in a way that
-                is sufficient to meet the SET Receiver's own reliability requirements,
+                The SET Recipient SHOULD ensure that the SET is persisted in a way that
+                is sufficient to meet the SET Recipient's own reliability requirements,
                 and MUST NOT expect or depend on a SET Transmitter to re-transmit or
-                otherwise make available to the SET Receiver a SET once the SET Receiver
+                otherwise make available to the SET Recipient a SET once the SET Recipient
                 acknowledges that it was received successfully.
             </t>
             <t>
-                Once the SET has been validated and persisted, the SET Receiver SHOULD
+                Once the SET has been validated and persisted, the SET Recipient SHOULD
                 immediately return a response indicating that the SET was successfully
-                delivered. The SET Receiver SHOULD NOT perform extensive business logic that
+                delivered. The SET Recipient SHOULD NOT perform extensive business logic that
                 processes the event expressed by the SET prior to sending this response.
                 Such logic SHOULD be executed asynchronously from delivery, in order to
                 minimize the expense and impact of SET delivery on the SET Transmitter.
             </t>
             <t>
                 The SET Transmitter SHOULD NOT re-transmit a SET, unless the response from
-                the SET Receiver in previous transmissions indicated a potentially recoverable
+                the SET Recipient in previous transmissions indicated a potentially recoverable
                 error (such as server unavailability that may be transient, or a decryption
-                failure that may be due to misconfigured keys on the SET Receiver's side).  In
+                failure that may be due to misconfigured keys on the SET Recipient's side).  In
                 the latter case, the SET Transmitter MAY re-transmit a SET, after an appropriate
-                delay to avoid overwhelming the SET Receiver (see <xref target="reliability"/>).
-            </t>
-            <t>
-                The SET Transmitter MAY provide an out-of-band mechanism by which a SET Receiver
```

Comment:
In the previous paragraph, we give guidance that discourages re-transmission of SETs in a number of circumstances. This paragraph is intended to clarify that this guidance does not preclude a transmitter making those SETs available through some other means (e.g. by providing a web console where receiver administrators could review SETs that failed). I thought it worth clarifying that this would not be against the spirit or letter of the spec, but I'm fine with removing it if others think this clarification is unnecessary.

Reply:
As it was written, it's the sort of text that will cause huge red flags during WG and IETF last calls and IESG review.  It describes functionality that the text makes it sound like may be necessary for some use cases but that that the standard doesn't provide.  We should either provide it (which I think would be going overboard) or not describe it, as it typically won't be necessary.

If WG members want something like it to be re-added after reviewing -03, then we can discuss that on its merits in Bangkok.  But it should be removed for now because the existing text creates more problems than it solves.